homeassistant/components/devolo_home_network/config_flow.py

```diff
@@ -21,7 +21,9 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-STEP_USER_DATA_SCHEMA = vol.Schema({vol.Required(CONF_IP_ADDRESS): str})
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {vol.Required(CONF_IP_ADDRESS): str, vol.Optional(CONF_PASSWORD, default=""): str}
```

Comment:
```suggestion
    {vol.Required(CONF_IP_ADDRESS): str, vol.Optional(CONF_PASSWORD): str}
```
you are setting to "", in 123 anyway